src/test/scala/mesosphere/marathon/LaunchQueueModuleTest.scala

```diff
@@ -121,18 +121,17 @@ class LaunchQueueModuleTest extends AkkaUnitTest with OfferMatcherSpec {
 
   class Fixture extends AutoCloseable {
     val app = MarathonTestHelper.makeBasicApp().copy(id = PathId("/app"))
+    val scheduledInstance = Instance.scheduled(app)
+    val task: Task = Task.provisioned(Task.Id.forInstanceId(scheduledInstance.instanceId), NetworkInfoPlaceholder(), app.version, Timestamp.now())
+    val mesosTask = MarathonTestHelper.makeOneCPUTask(task.taskId).build()
 
     val offer = MarathonTestHelper.makeBasicOffer().build()
-    val runspecId = PathId("/test")
-    val instance = TestInstanceBuilder.newBuilder(runspecId).addTaskWithBuilder().taskRunning().build().getInstance()
-    val task: Task = instance.appTask
 
-    val mesosTask = MarathonTestHelper.makeOneCPUTask(task.taskId).build()
-    val scheduledInstance = Instance.scheduled(app)
-    val launchTaskInstanceOp = LaunchTask(mesosTask, InstanceUpdateOperation.Provision(scheduledInstance), Some(scheduledInstance), Seq.empty)
+    val provisionOp = InstanceUpdateOperation.Provision(scheduledInstance.instanceId, AgentInfoPlaceholder(), app.version, Seq(Task.provisioned(Task.Id.forInstanceId(scheduledInstance.instanceId), NetworkInfoPlaceholder(), app.version, Timestamp.now())), Timestamp.now())
```

Comment:
Nit: I think this line is too long.  Also, why not reused the `task` from above?